Chat/assignment-chat/client-chat/src/main/java/ru/assignment/net/ChatClient.java

```diff
@@ -1,18 +1,22 @@
 package ru.assignment.net;
 
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.net.Socket;
 
-
-public class ChatClient {
-    private ClientConfiguration clientConfiguration;
+public class ChatClient implements DisconnectReceivedListener {
+    private final ClientConfiguration clientConfiguration;
     private Sender sender;
     private Receiver receiver;
     private Socket clientSocket;
+    public static Logger clientLogger ;
```

Comment:
Usual pattern to use logger is
private static final Logger LOG = LoggerFactory.getLogger(XXX.class);
Naming: LOG or log is debattable, it should be log, but some static analysis tools don't process the case correctly, so sometimes you have to use LOG